.kitchen.yml

```diff
@@ -16,7 +16,8 @@ platforms:
   - name: debian-8.4
   - name: windows
     driver:
-      box: opentable/win-2012r2-standard-amd64-nocm
+      box: mwrock/Windows2012r2
+      gui: false # This box apparently defaults to using a GUI.
```

Comment:
yeah its just the most commonly desired setting so I put it their to make that easy.